src/preact.d.ts

```diff
@@ -5,7 +5,9 @@ declare namespace preact {
 	}
 
 	interface PreactHTMLAttributes {
+		dangerouslySetInnerHTML?:Object;
```

Comment:
The effect will be the same, tho for organisational purposes it should be here.

maybe also better to define the type specifically with an interface? instead of allowing any object to be passed
this could be something like:
```
interface DangerouslySetInnerHTML {
 __html: string;
}
```